src/js/main.js

```diff
@@ -84,19 +86,54 @@ class Site {
         'left': randomLeft + 'vw',
         'transform': 'rotate(' + randomRotate + 'deg)',
       });
+
+      this.positionPostitDot();
     }
   }
 
-  stylePostitDot() {
+  positionPostitDot() {
     var _this = this;
 
-    if (_this.$postit.length) {
-      $('#postit-dot').css({
-        'top': '-' + (_this.dotDiameter / 2) + 'px',
+    _this.$postitDot.css({
+      'top': '-' + (_this.dotDiameter / 2) + 'px',
+    });
+  }
+
+  bindHoverDots() {
+    var _this = this;
+
+    if (_this.$hoverDotItem.length) {
+
+      _this.positionHoverDots();
+
+      _this.$hoverDotItem.on({
+        mouseenter: function(){
+          $(this).addClass('show-dot');
+        },
+        mouseleave: function(){
+          $(this).removeClass('show-dot');
+        },
       });
     }
   }
 
+  positionHoverDots() {
+    var _this = this;
+
+    var offset = _this.dotDiameter + 10;
+
+    _this.$hoverDot.each(function() {
+      var randomTop = _this.randomInt(100 - offset, offset); // min offset% from any edge
```

Comment:
when designing not defined stuff like this I'm very pro putting values like this `100` as set variables saved on the object/class